src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -112,7 +112,7 @@ bool ngraph::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ngrap
     // before CommonOptimization pipeline execution
     manager.register_pass<ngraph::pass::MOCTransformations>(true, false);
 
-    manager.register_pass<ov::pass::ConvertCompressedOnlyToLegacy, false>();
+    manager.register_pass<ov::pass::ConvertCompressedOnlyToLegacy>();
```

Comment:
I think it makes sense to have a comment that in majority of plugins IRv11 in FP16 is not supported but CPU and GNA plugins. So we have to convert IRv11 in FP16 to IRv10